.github/workflows/building.yml

```diff
@@ -0,0 +1,129 @@
+name: Building Wheels
+
+on:
+  push:
+    tags:
+      - '*'
+
+jobs:
+
+  wheel:
+    runs-on: ${{ matrix.os }}
```

Comment:
Let's just run on `ubuntu-latest`.

Reply:
If we want to support `mac-os` in the future, we should probably just create a new workflow.